keel-orca/src/main/kotlin/com/netflix/spinnaker/keel/orca/OrcaIntentLauncher.kt

```diff
@@ -44,30 +49,30 @@ open class OrcaIntentLauncher
       val result = intentProcessor(intentProcessors, intent).converge(intent)
 
       if (result.orchestrations.isEmpty()) {
-        log.info("Not converging state for intent (id: ${intent.id}, reason: ${result.reason})")
+        log.info("Not converging state for intent ${intent.id}, ${result.changeSummary}")
       } else {
         applicationEventPublisher.publishEvent(ConvergenceRequiredEvent(intent))
       }
 
       val orchestrationIds = result.orchestrations.map {
-        log.info("Launching orchestration for intent (id: ${intent.id}, reason: ${result.reason})")
+        log.info("Launching orchestration for intent ${intent.id}, ${result.changeSummary}")
```

Comment:
Oops. This log (and the one above) should get `StructuredArguments`-ified.

Reply:
Fixed